libtenzir/builtins/formats/json.cpp

```diff
@@ -1660,20 +1113,79 @@ class read_json_plugin final
         result = failure::promise();
       }
     }
+
+    TRY(result);
+    return std::make_unique<parser_adapter<json_parser>>(
+      json_parser{std::move(args)});
+  }
+};
+
+class read_gelf_plugin final
+  : public virtual operator_plugin2<parser_adapter<json_parser>> {
+public:
+  auto name() const -> std::string override {
+    return "read_gelf";
+  }
+  auto
+  make(invocation inv, session ctx) const -> failure_or<operator_ptr> override {
```

Comment:
Unresolving this, because it's still the same.

By the way: I don't even care that much about this specific line, but this is just a reminder to try stick to the coding style. This also applies to comments, template arguments, max line length... I think some of your commits were done without auto formatting, so you go beyond the line limit etc. Maybe at the very end you can go over this once more, adjust the particularly jarring coding-style violations (and leave the not so jarring ones as is to save time), and then auto-format everything in the PR diff once more (perhaps after a squash).